docs/Developing.md

```diff
@@ -132,14 +132,81 @@ or visit `http://localhost:3000/ui/` direct.
 
 xpcshell and mochitest
 ----------------------
-
 These are tests for the [chrome](https://developer.mozilla.org/docs/Glossary/Chrome)
 modules and files.
 
-Currently these are not runnable in the github repository, and an export to
-mozilla-central is required. We are
-[working on improving](https://bugzilla.mozilla.org/show_bug.cgi?id=1238570) this
-situation.
+**Test Export to Mozilla Central(gecko-dev)**
+
+The gecko-dev repo is cloned locally and is used as the xpcshell and 
```

Comment:
New developers coming from Firefox won't necessarily know what gecko-dev is. I think you should just reference `The mozilla-central (or an integration branch) repo`.

Maybe add a `See [the simple Firefox build instructions](https://developer.mozilla.org/docs/Mozilla/Developer_guide/Build_Instructions/Simple_Firefox_build) for setting up a repository for a Firefox build. There is also a [git-cinnibar](https://github.com/glandium/git-cinnabar) wrapper to allow using git for the repository.`